src/Contact/View/page.twig

```diff
@@ -1,18 +1,12 @@
 {% include 'Common/View/header.twig' %}
 <section class="container contact-section">
-	<div class="form">
-		<h1 input class="form-title">NAPISZ DO NAS</h1>
-		<form method="post" action="">
-			<input class="form-input" type="text" name="name" placeholder="Imię i nazwisko">
-			<input class="form-input" type="email" name="email" placeholder="E-mail">
-			<input class="form-input" type="text" name="topic" placeholder="Temat">
-			<textarea id="message" class="form-input form-input--text" name="text" placeholder="Treść wiadomości"></textarea>
-			<div class="u-center-text">
-				<button class="submit-button" type="submit">
-					Wyślij
-				</button>
-			</div>
-		</form>
-	</div>
+        <form method="post" action="" class="form">
+<h1 class="form-title">NAPISZ DO NAS</h1> <br>
```

Comment:
If you want to add space between items, you'd better use the css property "margin" instead of "<br>". Where do you think you can set the margin value to make this break?

Reply:
I deleted action="", seems that the margins are all right in contact-tasks.css file.